### ISSUE TITLE
Can't import the named export 'getCurrentInstance' from non EcmaScript module

### ISSUE THREAD

**Issue by BulatSa** [uNmAnNeR/imaskjs#898]
**Describe the bug**
Can't import the named export 'getCurrentInstance' from non EcmaScript module (only default export is available)
![image](https://github.com/uNmAnNeR/imaskjs/assets/8298469/ec5a4f16-b70a-4070-9220-0ad9942d21b3)

**To Reproduce**
I use in project Nuxt 2.17.0, Vue 2.7.14, NodeJs 18.15

**Environment:**
 - OS: Win10
 - IMask version 7.0.1
 - Framework/plugin version if used - Nuxt 2.17.0, Vue 2.7.14

**Comment by BulatSa** [comment]
Solution for for NuxtJS:

```
// nuxt.config.js
export default {
  build: {
    extend(config) {
      config.module.rules.push({
        test: /\.mjs$/,
        include: /node_modules/,
        type: 'javascript/auto',
      })
    },
  },
}
```

Solution for for Vue2:
```
// vue.config.js
module.exports = {
  configureWebpack: {
    module: {
      rules: [
        {
          test: /\.mjs$/,
          include: /node_modules/,
          type: "javascript/auto"
        }
      ] 
    }
  }
}
```

**Comment by uNmAnNeR** [comment]
related links:
https://github.com/vuejs/pinia/issues/675

https://github.com/vueuse/vue-demi/issues/171

@uNmAnNeR check `exports` here to help a bundler:
https://github.com/vuejs/pinia/blob/v2/packages/pinia/package.json

**Comment by uNmAnNeR** [comment]
should be fixed in v7.1.1
